build/debs/BUILD

```diff
@@ -169,7 +169,7 @@ k8s_deb(
     depends = [
         "kubelet (>= 1.8.0)",
         "kubectl (>= 1.8.0)",
-        "kubernetes-cni (>= 0.6.0)",
+        "kubernetes-cni (>= %s)" % CNI_VERSION,
```

Comment:
is this the right thing to do? e.g. cri-tools is specifying a different version than CRI_TOOLS, and kubelet/kubectl are quite a bit behind. but maybe those are wrong?

Reply:
this will result in the same value today but de-duped. wether we continue to specify the minimum version or not I'm not sure. IIRC, technically 0.5.0 would function today, just with some reduction in capabilities. I'm not sure how we want to manage this, but so far it seems like we use the version we are shipping.

Reply:
Yes, if we ever want to not require the latest version we test with for a dep like this, we'll change this but for now this reflects reality